puppeteer-login-script.js

```diff
@@ -115,3 +117,25 @@ const createCollections = async function(context, page) {
     console.log(e);
   }
 };
+
+const createExplorations = async function(context, page) {
+  try {
+    // eslint-disable-next-line no-console
+    console.log('Creating Exploration...');
+    // Load in Exploration
+    // eslint-disable-next-line dot-notation
+    await page.goto('http://127.0.0.1:8181/admin');
+    await page.waitFor(2000);
+    await page.evaluate('window.confirm = () => true');
+    await page.screenshot({path: 'buddy-screenshot.png'});
+    await page.click(
+      'div:nth-child(2) > span.col-lg-2.col-md-2.col-sm-2.ml-auto > button');
```

Comment:
What does this mean, could you point to this element in some other way?

Reply:
This is supposed to point to the reload exploration button in the admin activities tab. I think I found a better way to point to it via class.